lib/sidekiq/statistic/views/styles/sidekiq-statistic-dark.css

```diff
@@ -0,0 +1,126 @@
+/* === DARK MODE STYLES === */
+
+/* === CONTAINER === */
+
+.statistic__container {
+  background-color: #282828;
+}
+
+/* === DATEPICKER === */
+
+.statistic__datepicker {
+  border-bottom: 1px solid #333333;
+}
+
+.statistic__datepicker input {
+  background: #333333;
+  border: 1px solid #555;
+  box-shadow: 0 0 5px rgba(255, 255, 255, 0.1);
+}
+
+#statistic__search {
+  background: #333333;
+  border: 1px solid #555;
+  box-shadow: 0 0 5px rgba(255, 255, 255, 0.1);
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+}
+
+/* === UI DATEPICKER jQuery === */
+
+.ui-datepicker {
+  background-color: #282828;
+  border: 1px solid #d5d5d5;
+}
+
+.ui-datepicker-today {
+  background: #CD3844;
+}
+
+.ui-datepicker a:active, a:hover, a:visited {
+  color: #ccc;
+}
+
+/* === TABS === */
+
+.nav-tabs {
+  background-color: #333333;
+  border-bottom: 1px solid #333333;
+}
+
+.nav-tabs > li.active > a {
+  background-color: #282828;
+  border: 1px solid #282828;
+  color: #DCDCDC;
+}
+
+.nav-tabs > li > a:hover, a:visited {
+  background-color: #4A4443;
+  border: 1px solid #4A4443;
+  color: #DCDCDC;
+  cursor: pointer;
+}
+
+.nav-tabs > li.active > a:hover {
+  background-color: #4A4443;
+  border: 1px solid #4A4443;
+  color: #DCDCDC;
+  cursor: pointer;
+}
+
+.nav-tabs .statistic__tab {
+  background-color: rgb(64, 58, 57);
+}
+
+/* === Worker table TOGGLE === */
+
+.worker__toggle-visibility {
+  color: rgb(213, 213, 213) !important;
+}
+
+.worker__toggle-visibility {
+  background: inherit;
+  border: 1px solid rgb(102, 102, 102);
+}
```

Comment:
This both can be combined.
The `!important` is really necessary?
Maybe this si not getting the color properly due to a selector specificity. You can check it using the Chrome Dev Tools to see.